test/expr/cast/as_coerce.swift

```diff
@@ -163,3 +163,28 @@ let _: Int = any as Int // expected-error {{'Any' is not convertible to 'Int'}}
 // expected-note@-1 {{did you mean to use 'as!' to force downcast?}} {{18-20=as!}}
 let _: Int? = any as Int // expected-error {{'Any' is not convertible to 'Int'}}
 // expected-note@-1 {{did you mean to use 'as?' to conditionally downcast?}} {{19-21=as?}}
+
+// https://github.com/apple/swift/issues/63926
+
+func fn63926(_: Int) {} // expected-note {{found candidate with type '(Int) -> ()'}}
```

Comment:
Since all we check here are functions, let's wrap all the new tests in a `do { ... }` and drop `63926` from the names.

Reply:
Fixed!